app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitAttendeeNotesApiController.php

```diff
@@ -0,0 +1,228 @@
+        $this->repository = $attendee_notes_repository;
+        $this->attendee_service = $attendee_service;
+    }
+
+    /**
+     * @param $summit_id
+     * @param $attendee_id
+     * @return mixed
+     * @throws \HTTP401UnauthorizedException
+     */
+    public function getAttendeeNotes($summit_id, $attendee_id)
+    {
+        $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->getResourceServerContext())->find($summit_id);
+        if (is_null($summit)) return $this->error404();
+
+        $attendee = $this->attendee_repository->getById(intval($attendee_id));
+        if (is_null($attendee)) return $this->error404();
+
+        return $this->_getAll(
+            function () {
+                return [
+                    'ticket_id' => ['=='],
+                    'author_fullname' => ['=@', '==', '@@'],
+                    'author_email' => ['=@', '==', '@@'],
+                    'created' => ['==', '>=', '<=', '>', '<'],
+                    'edited' => ['==', '>=', '<=', '>', '<'],
+                ];
+            },
+            function () {
+                return [
+                    'ticket_id' => 'sometimes|integer',
+                    'author_fullname' => 'sometimes|string',
+                    'author_email' => 'sometimes|string',
+                    'created' => 'sometimes|required|date_format:U',
+                    'edited' => 'sometimes|required|date_format:U',
+                ];
+            },
+            function () {
+                return [
+                    'id',
+                    'created'
+                ];
+            },
+            function ($filter) {
+                return $filter;
+            },
+            function () {
+                return SerializerRegistry::SerializerType_Private;
+            },
+            null,
+            null,
+            function ($page, $per_page, $filter, $order, $applyExtraFilters) use ($attendee) {
+                return $this->repository->getAllBySummitAttendee
+                (
+                    $attendee,
+                    new PagingInfo($page, $per_page),
+                    call_user_func($applyExtraFilters, $filter),
+                    $order
+                );
+            }
+        );
+    }
+
+    /**
+     * @param $summit_id
+     * @param $attendee_id
+     * @param $note_id
+     * @return mixed
+     */
+    public function getAttendeeNote($summit_id, $attendee_id, $note_id) {
+        return $this->processRequest(function() use($summit_id, $attendee_id, $note_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $attendee = $this->attendee_repository->getById(intval($attendee_id));
+            if (is_null($attendee)) return $this->error404();
+
+            if ($attendee->getSummit()->getId() != intval($summit_id))
```

Comment:
this is redundant get attendee from summit

Reply:
done